diff -urN cdparanoia-III-alpha9.8-old/interface/utils.h cdparanoia-III-alpha9.8/interface/utils.h
--- cdparanoia-III-alpha9.8-old/interface/utils.h	2004-07-08 20:51:39.058443048 +0000
+++ cdparanoia-III-alpha9.8/interface/utils.h	2004-07-08 20:53:39.825083728 +0000
@@ -112,6 +112,7 @@
       break;
     case CDDA_MESSAGE_FORGETIT:
     default:
+      ;
     }
   }
 }
@@ -127,6 +128,7 @@
       break;
     case CDDA_MESSAGE_FORGETIT:
     default:
+      ;
     }
   }
 }
@@ -169,6 +171,7 @@
       break;
     case CDDA_MESSAGE_FORGETIT:
     default:
+      ;
     }
   }
   if(malloced)free(buffer);
@@ -205,6 +208,7 @@
       break;
     case CDDA_MESSAGE_FORGETIT:
     default:
+      ;
     }
   }
   if(malloced)free(buffer);
