diff -urN cdparanoia-III-alpha9.8.org/interface/Makefile.in cdparanoia-III-alpha9.8/interface/Makefile.in
--- cdparanoia-III-alpha9.8.org/interface/Makefile.in	1999-12-14 05:27:59.000000000 +0100
+++ cdparanoia-III-alpha9.8/interface/Makefile.in	2006-10-12 21:45:41.979894750 +0200
@@ -46,7 +46,7 @@
 	$(RANLIB) libcdda_interface.a
 
 libcdda_interface.so: 	$(OFILES)	
-	$(CC) -fpic -shared -o libcdda_interface.so.0.$(VERSION) -Wl,-soname -Wl,libcdda_interface.so.0 $(OFILES)
+	$(CC) -fpic -shared -o libcdda_interface.so.0.$(VERSION) -Wl,-soname -Wl,libcdda_interface.so.0 $(OFILES) -lm
 	[ -e libcdda_interface.so.0 ] || ln -s libcdda_interface.so.0.$(VERSION) libcdda_interface.so.0
 	[ -e libcdda_interface.so ] || ln -s libcdda_interface.so.0.$(VERSION) libcdda_interface.so
 
diff -urN cdparanoia-III-alpha9.8.org/paranoia/Makefile.in cdparanoia-III-alpha9.8/paranoia/Makefile.in
--- cdparanoia-III-alpha9.8.org/paranoia/Makefile.in	1999-12-14 05:28:04.000000000 +0100
+++ cdparanoia-III-alpha9.8/paranoia/Makefile.in	2006-10-12 21:48:24.118027750 +0200
@@ -44,7 +44,7 @@
 	$(RANLIB) libcdda_paranoia.a
 
 libcdda_paranoia.so: 	$(OFILES)	
-	$(CC) -fpic -shared -o libcdda_paranoia.so.0.$(VERSION) -Wl,-soname -Wl,libcdda_paranoia.so.0 $(OFILES)
+	$(CC) -fpic -shared -o libcdda_paranoia.so.0.$(VERSION) -Wl,-soname -Wl,libcdda_paranoia.so.0 $(OFILES) -L../interface -lcdda_interface -lm
 	[ -e libcdda_paranoia.so.0 ] || ln -s libcdda_paranoia.so.0.$(VERSION) libcdda_paranoia.so.0
 	[ -e libcdda_paranoia.so ] || ln -s libcdda_paranoia.so.0.$(VERSION) libcdda_paranoia.so
 
